service/controller/app/resource/releasemigration/create.go

```diff
@@ -38,17 +38,10 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return nil
 	}
 
-	//TODO: Remove this statement when we need to test on control planes.
-	if key.InCluster(cr) {
-		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("app %#q uses InCluster kubeconfig no need to migrate releases", key.AppName(cr)))
-		r.logger.LogCtx(ctx, "level", "debug", "message", "cancelling the resource")
-		return nil
-	}
```

Comment:
Removing this as I don't think we need it anymore.

Reply:
yeah, it only reconciled on chart-operator app CR.